### PR TITLE
Added localization for debug level selector

### LINKED ISSUE #11973: Debug console severity selector is not localized
### Bug Description:

Debug severity drop down selector is not localized.

### Steps to Reproduce:

1. Open Theia
2. Install any language pack
3. Reload page (now interface should be in chosen language)
4. Open debug console
5. On the right side of panel there will be selector, but in English

### Additional Information

Message is probably located [here](https://github.com/eclipse-theia/theia/blob/master/packages/core/src/common/severity.ts#L33)

image

### PR DESCRIPTION
**Pull request by xcariba**
#### What it does

Fixes #11973.

Adds localization support for debug console severity.

#### How to test

* Start theia and open folder
* Create js file and write different log calls (`console.error('My Error'); console.info('My Info');`
* Add `debugger;` call or set breakpoint
* Start debugger for created js file
* Open debug console view
* Write some js call and press enter (in my case debug console was empty until some call was made from console)
* Switch debug levels, it will show messages with chosen level

#### Review checklist

- [ ] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/core/src/common/severity.ts [hunk 1]
```diff
@@ -30,28 +31,28 @@ export enum Severity {
...
-    const error = 'Errors';
-    const warning = 'Warnings';
-    const info = 'Info';
-    const log = 'Log';
-    const ignore = 'All';
+    const error = nls.localize('theia/core/severity/errors', 'Errors');
+    const warning = nls.localize('theia/core/severity/warnings', 'Warnings');
+    const info = nls.localize('theia/core/severity/info', 'Info');
+    const log = nls.localize('theia/core/severity/log', 'Log');
+    const ignore = nls.localize('theia/core/severity/all', 'All');
```
Comment (msujew) [inline packages/core/src/common/severity.ts:38]: Suggestion: I'm not too sure whether we should localize these values directly. I would rather have a `Severity.toLocaleString()` function that accepts a `string | Severity` and returns a localized version of the string. Use the result at the caller site of `debug-console-contribution.tsx`.

**Comment by msujew** [comment]
@xcariba Are you still interested in contributing this change?

**Comment by xcariba** [comment]
@msujew Yes, I just don't have enough time =( I agree with your suggestion, fixed PR

**Review by msujew — APPROVED** [review]
Looks good to me, thanks 👍

**Pull request merged**